test/schema.test.ts

```diff
@@ -33,24 +33,14 @@ describe("commands/schema", function () {
   test("throws with missing table", async function () {
     const consoleError = spy(console, "error");
     await yargs(["schema", "ignored_31337_99"]).command(mod).parse();
-    assert.calledWith(consoleError, "Failed to get schema from table");
+    assert.calledWith(consoleError, "Not Found");
   });
 
-  test("passes with local-tableland", async function () {
+  test("Schema passes with local-tableland", async function () {
     const consoleLog = spy(console, "log");
     await yargs(["schema", "healthbot_31337_1"]).command(mod).parse();
-    assert.calledWith(
```

Comment:
ah wait, ignore my above statement, you've changed things so that we're logging the object, not the string?

Reply:
This _is_ nice, though see my comment above about deeply nested objects.